src/lambdas/initiateDataRequest/zendeskTicketDiffersFromRequest.ts

```diff
@@ -69,6 +69,40 @@ const getZendeskCustomSpaceSeparatedStringAsArray = (
   return mapSpaceSeparatedStringToList(fieldValue)
 }
 
+const datesMatch = (
+  ticketDetails: ZendeskTicket,
+  requestParams: DataRequestParams
+) => {
+  const ticketLegacyDateField = getZendeskCustomFieldValue(
+    ticketDetails,
+    getEnvAsNumber('ZENDESK_FIELD_ID_DATE_FROM')
+  )
+  const ticketDates = getZendeskCustomSpaceSeparatedStringAsArray(
+    ticketDetails,
+    getEnvAsNumber('ZENDESK_FIELD_ID_DATES')
+  )
+  if (ticketLegacyDateField) {
+    const returnValue = matchArrayParams(
+      [ticketLegacyDateField as string],
```

Comment:
it might be more readable to do `as string | null` when we create ticketLegacyDateField so it's clear we know it's that because it's a single date rather than casting here.

Reply:
I like it. Done!